boot/zephyr/include/mcuboot_config/mcuboot_config.h

```diff
@@ -73,6 +73,10 @@
 #define MCUBOOT_SWAP_USING_MOVE 1
 #endif
 
+#ifdef CONFIG_BOOT_SWAP_USING_SCRATCH
+#define MCUBOOT_SWAP_USING_SCRATCH 1
```

Comment:
There's no need to have this, this mode is already the default.

Reply:
For example I want to have something like this:
```
#ifdef MCUBOOT_SWAP_USING_SCRATCH
...
#endif
```
Now I have to bring in the boot/bootutil/src/bootutil_priv.h to have this defined, or bring the logic that "defines" it, to an unit where I try to do that code.
So maybe the logic from boot/bootutil/src/bootutil_priv.h should be here to auto-magically define the `MCUBOOT_SWAP_USING_SCRATCH`.

Reply:
Sure, sounds good. You'll probably want to make it explicit for all ports then, also for the simulator.